packages/notifi-react-card/lib/components/subscription/SubscriptionCardV1.tsx

```diff
@@ -49,6 +52,23 @@ export const SubscriptionCardV1: React.FC<SubscriptionCardV1Props> = ({
           copy={{ label: inputLabels?.email }}
         />
       ) : null}
+      {inputSeparators?.first?.content ? (
+        <div
+          className={clsx(
+            'NotifiInputSeparator__container',
+            inputSeparators?.first?.classNames?.container,
+          )}
+        >
+          <div
+            className={clsx(
+              'NotifiInputSeparator__content',
+              inputSeparators.first.classNames?.content,
+            )}
+          >
+            {inputSeparators?.first?.content}
+          </div>
+        </div>
+      ) : null}
```

Comment:
hmmm, having `inputSeparator.first / second` makes it a little unwieldy to use i think -- if I have email disabled, for example, where should I expect "first" separator to go? 

I'd probably rather see some concept of "email.footer / sms.footer / telegram.footer" instead, because that makes it feel more related to each input field (and makes it clear where the "separator" will go

Reply:
ooo, great point. Let me update that

Reply:
updated it so it looks like this.
i might improve this to also handle errors or something.
<img width="404" alt="image" src="https://user-images.githubusercontent.com/105258726/194722316-9a39c2ea-ef6d-4e6e-802f-1b05688151b1.png">